api/v1alpha1/tritoninterfaceserver_types.go

```diff
@@ -42,6 +42,11 @@ type TritonInterfaceServerSpec struct {
 
 	// +kubebuilder:validation:Optional
 	PodResources Resource `json:"podResources"`
+
+	// TLSSpec includes the TLS Secret Name (optional)
+	//  +kubebuilder:validation:Optional
+	TlsSecretName string `json:"tlsSecretName,omitempty"` // The TLS Secret Name (optional)
```

Comment:
comments should be above the variable definition. Including both inline and comment above makes the code unclean and readability is reduced